README.md

```diff
@@ -100,7 +108,8 @@ For clarity, an example of the setup below can be found in the [ENS subgraph rep
        --ipfs /ip4/127.0.0.1/tcp/5001 \
        --subgraph-name your-subgraph
     ```
-
+    _Note: if you are deploying to a hosted node use the `-auth` flag
```

Comment:
Let's rephrase this. Hosted nodes are not really a thing; we only set them up for hackathons so far. How about?

```
_Note: If the Graph Node you are deploying to requires authorization, make sure
to authorize with the node using e.g. 'graph auth http://127.0.0.1:8020 <access token>`
before deploying.
```

Let's not mention the `--auth` flag here, this isn't how it was supposed to work.